### PR TITLE
refactored word2vec

### PR DESCRIPTION
**Pull request by oldosamaibraheem11**
Change list:

Word2Vec will work now for file that is one line per sentence 
Kmeans doesn't use null values now

### CODE REVIEW

File: word2vec.py [hunk 1]
```diff
@@ -13,9 +13,9 @@ def __iter__(self):
...
         for line in open(CorpusPath):
             yield utils.simple_preprocess(line)
-
 PathToFile = input("Enter the path to the corpus: ")
 CorpusName = input("What is the name of your corpus (this will be the name of model)? ")
 AllSentences = MyCorpus(PathToFile)
 model = Word2Vec(sentences=AllSentences, window=5, min_count=2, workers=8,size=248, iter=25)
-model.save(f"models\\{CorpusName.lower()}.model")
+model.save(f"models/{CorpusName.lower()}.model")
+print(model.wv.most_similar('son'))
```
Comment (pnklshh) [inline word2vec.py]: Can we remove the print statement used testing ?

**Review by pnklshh — CHANGES_REQUESTED** [review]
Looks good...just remove the print statement.

**Comment by pnklshh** [comment]
@osamaibraheem11 Will the database still have null vector values for some of the sentences after this change?

**Comment by oldosamaibraheem11** [comment]
> @osamaibraheem11 Will the database still have null vector values for some of the sentences after this change?

@pnklshh out of the 16 million words, 499 are returning none

**Pull request merged**